server/eclipse-project/src/main/java/us/freeandfair/corla/endpoint/AbstractEndpoint.java

```diff
@@ -24,6 +24,7 @@
 import org.eclipse.jetty.http.HttpStatus;
 import org.hibernate.HibernateException;
 
+import org.hibernate.service.spi.InjectService;
```

Comment:
Is this here so it will be inherited by a child class or is it just an artifact of some no longer present changes?

Reply:
Huh. I think that might just be IntelliJ being overzealous. I didn't (intend to) change AbstractEndpoint - I think I can just delete that (again).